core/app/components/FormBuilder/SaveFormButton.tsx

```diff
@@ -18,7 +21,7 @@ export const SaveFormButton = ({ form, className, disabled }: Props) => {
 			form={form}
 			type="submit"
 			data-testid="save-form-button"
-			disabled={disabled}
+			disabled={disabled != null ? disabled : !isChanged}
```

Comment:
maybe i should just remov the `disabled` prop entirely and make it `!isChanged` by default